tsconfig.json

```
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "declaration": false,
+        "declaration": true,
         "outDir": "build",
         "noImplicitAny": false,
```

If you've converted it all to typescript then you can set this to true
```suggestion
        "noImplicitAny": true,
```
and also remove the checkJs/allowJs below

---

Going for `"strict": true,` instead since it enables this as well.